### ISSUE TITLE
Unable to find generated LoRA file

### ISSUE THREAD

**Issue by weiwei83** [bmaltais/kohya_ss#2327]
<img width="1125" alt="image" src="https://github.com/bmaltais/kohya_ss/assets/10830200/689639f8-a0fb-41e1-8c54-e9fc42fd06cc">

**Comment by weiwei83** [comment]
<img width="817" alt="image" src="https://github.com/bmaltais/kohya_ss/assets/10830200/bbaf8cb4-679f-484b-8ec8-ce9b0cb92af9">

**Comment by weiwei83** [comment]
bucket_no_upscale = true
bucket_reso_steps = 64
cache_latents = true
cache_latents_to_disk = true
caption_dropout_every_n_epochs = 0
caption_dropout_rate = 0
caption_extension = ".txt"
clip_skip = 1
dynamo_backend = "no"
enable_bucket = true
epoch = 100
gradient_accumulation_steps = 1
gradient_checkpointing = true
huber_c = 0.1
huber_schedule = "snr"
keep_tokens = 1
learning_rate = 0.0001
logging_dir = " /data/train/ssyy/log"
loss_type = "l2"
lr_scheduler = "constant"
lr_scheduler_args = []
lr_scheduler_num_cycles = 1
lr_scheduler_power = 1
lr_warmup_steps = 0
max_bucket_reso = 2048
max_data_loader_n_workers = 0
max_grad_norm = 1
max_timestep = 900
max_token_length = 75
max_train_steps = 3975
min_bucket_reso = 256
min_snr_gamma = 5
min_timestep = 100
mixed_precision = "fp16"
multires_noise_discount = 0
network_alpha = 64
network_args = []
network_dim = 128
network_dropout = 0
network_module = "networks.lora"
no_half_vae = true
noise_offset_type = "Original"
optimizer_type = "AdamW"
optimizer_args = []
output_dir = " /data/train/ssyy/model"
output_name = "LoRA-ssyy-sdxl-v2"
pretrained_model_name_or_path = "stabilityai/stable-diffusion-xl-base-1.0"
prior_loss_weight = 1
resolution = "1024,1024"
sample_every_n_epochs = 5
sample_prompts = " /data/train/ssyy/model/prompt.txt"
sample_sampler = "euler"
save_every_n_epochs = 10
save_model_as = "safetensors"
save_precision = "fp16"
scale_weight_norms = 0
seed = 12345
text_encoder_lr = 0.0001
train_batch_size = 4
train_data_dir = " /data/train/ssyy/img"
training_comment = "trigger: yeonhwa"
unet_lr = 0.0001
xformers = true

**Comment by weiwei83** [comment]
<img width="591" alt="image" src="https://github.com/bmaltais/kohya_ss/assets/10830200/7cc254c1-5cf8-4348-b90c-3edd93a45638">

**Comment by weiwei83** [comment]
<img width="1238" alt="image" src="https://github.com/bmaltais/kohya_ss/assets/10830200/404576d3-9d5c-4454-b840-64c40a3c180e">
<img width="1141" alt="image" src="https://github.com/bmaltais/kohya_ss/assets/10830200/bce413cd-46d6-46de-a1ed-d4790a877ade">

**Comment by weiwei83** [comment]
When I reinstalled the entire project, new problems appeared

<img width="758" alt="image" src="https://github.com/bmaltais/kohya_ss/assets/10830200/104e9c16-d4cd-4d32-90fa-3e25ef7bc182">
<img width="333" alt="image" src="https://github.com/bmaltais/kohya_ss/assets/10830200/896d9886-cc36-4af8-9bf8-fe4144ca1a0d">

**Comment by weiwei83** [comment]
The problem was solved. I reinstalled the project and then performed the operations in "prepare training data". The program recognized all the paths and started working.

**Comment by weiwei83** [comment]
I only have 25 pictures, but the program determines that I have 50 pictures
<img width="1054" alt="image" src="https://github.com/bmaltais/kohya_ss/assets/10830200/bb46dd88-2d68-4592-9018-cfa6ea5e7198">
